website/docs/plugins/submitit_launcher.md

```diff
@@ -35,23 +35,28 @@ Submitit actually implements 2 different launchers: `submitit_slurm` to run on a
 You can discover the SLURM Launcher parameters with:
 ```yaml title="$ python your_app.py hydra/launcher=submitit_slurm --cfg hydra -p hydra.launcher"
 # @package hydra.launcher
-_target_: hydra_plugins.hydra_submitit_launcher.submitit_launcher.SlurmLauncher
 submitit_folder: ${hydra.sweep.dir}/.submitit/%j
 timeout_min: 60
-cpus_per_task: 1
-gpus_per_node: 0
+cpus_per_task: null
```

Comment:
is this mandatory ? Do users have to type `cpus_per_task: null` ?
It feels really hostile.
I think a minimal example should only have `mem_gb`, and timeout_min set.

Reply:
This isn't an example and they don't have to explicitly type `null` this is just showing what happens if you dump the config node in order to discover the possible values and their defaults